docs/schema/azure.md

```diff
@@ -845,3 +871,391 @@ Representation of an [AzureStorageBlobContainer](https://docs.microsoft.com/en-u
+|enablecassandraconnector | Enables the cassandra connector on the Cosmos DB C* account.|
+|connectoroffer | The cassandra connector offer type for the Cosmos DB database C* account.|
+|disablekeybasedmetadatawriteaccess | Disable write operations on metadata resources (databases, containers, throughput) via account keys.|
+|keyvaulturi | The URI of the key vault.|
+|enablefreetier | Flag to indicate whether Free Tier is enabled.|
+|enableanalyticalstorage | Flag to indicate whether to enable storage analytics.|
+|defaultconsistencylevel | The default consistency level and configuration settings of the Cosmos DB account.|
+|maxstalenessprefix | When used with the Bounded Staleness consistency level, this value represents the number of stale requests tolerated.|
+|maxintervalinseconds | When used with the Bounded Staleness consistency level, this value represents the time amount of staleness (in seconds) tolerated.|
+
+### Relationships
+
+- Azure Subscription contains one or more database accounts.
+
+        ```
+        (AzureSubscription)-[RESOURCE]->(AzureCosmosDBAccount)
+        ```
+- Azure Database Account has write permissions from, read permissions from and is associated with Azure CosmosDB Locations.
+
+        ```
+        (AzureCosmosDBAccount)-[WRITE_PERMISSIONS_FROM]->(AzureCosmosDBLocation)
+        ```
+        (AzureCosmosDBAccount)-[READ_PERMISSIONS_FROM]->(AzureCosmosDBLocation)
+        ```
+        (AzureCosmosDBAccount)-[ASSOCIATED_WITH]->(AzureCosmosDBLocation)
+        ```
+- Azure Database Account contains one or more Cors Policy.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBCorsPolicy)
+        ```
+- Azure Database Account contains one or more failover policies.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBAccountFailoverPolicy)
+        ```
+- Azure Database Account is configured with one or more private endpoint connections.
+
+        ```
+        (AzureCosmosDBAccount)-[CONFIGURED_WITH]->(AzureCDBPrivateEndpointConnection)
+        ```
+- Azure Database Account is configured with one or more virtual network rules.
+
+        ```
+        (AzureCosmosDBAccount)-[CONFIGURED_WITH]->(AzureCosmosDBVirtualNetworkRule)
+        ```
+- Azure Database Account contains one or more SQL databases.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBSqlDatabase)
+        ```
+- Azure Database Account contains one or more Cassandra keyspace.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBCassandraKeyspace)
+        ```
+- Azure Database Account contains one or more MongoDB Database.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBMongoDBDatabase)
+        ```
+- Azure Database Account contains one or more table resource.
+
+        ```
+        (AzureCosmosDBAccount)-[CONTAINS]->(AzureCosmosDBTableResource)
+        ```
+
+## AzureCosmosDBLocation
```

Comment:
Can you please add a link to the Azure API resource that this is based off of if possible? It's helpful to trace back.

Reply:
Okay! I'll add the generic link that @ramonpetgrave64 suggested since Microsoft keeps updating their documentation and the link might break in the future.